exercises/practice/saddle-points/create_test_table.sql

```diff
@@ -0,0 +1,27 @@
+DROP TABLE IF EXISTS tests;
+CREATE TABLE IF NOT EXISTS tests (
+    -- uuid and description are taken from the test.toml file
+    uuid TEXT PRIMARY KEY,
+    description TEXT NOT NULL,
+    -- The following section is needed by the online test-runner
+    status TEXT DEFAULT 'fail',
+    message TEXT,
+    output TEXT,
+    test_code TEXT,
+    task_id INTEGER DEFAULT NULL,
+    -- Here are columns for the actual tests
+    matrix TEXT NOT NULL,     -- json array of arrays
+    expected TEXT NOT NULL    -- json array of objects
+);
+
+INSERT INTO tests (uuid, description, matrix, expected)
+VALUES
+        ('3e374e63-a2e0-4530-a39a-d53c560382bd', 'Can identify single saddle point', '[[9,8,7],[5,3,2],[6,6,7]]', '[{"row":2,"column":1}]'),
```

Comment:
This block uses 8-space indents. The prior block uses 4-space. The next file uses 2-space.